test/alternator/test_manual_requests.py

```diff
@@ -174,6 +174,18 @@ def test_tags_return_empty_body(dynamodb, test_table):
     response = requests.post(req.url, headers=req.headers, data=req.body, verify=False)
     assert not response.text
 
+def test_create_table_without_keyschema(dynamodb, test_table_s):
+    payload = '{"TableName": "alternator_Test1", "AttributeDefinitions": [{"AttributeName": "lock_key", "AttributeType": "S"}, {"AttributeName": "sort_key", "AttributeType": "S"}], "BillingMode": "PAY_PER_REQUEST"}'
+    req = get_signed_request(dynamodb, 'CreateTable', payload)
+    response = requests.post(req.url, headers=req.headers, data=req.body, verify=False)
+    assert "ValidationException" in response.text and "Missing KeySchema member" in response.text
+
+def test_create_table_without_attribute_definition(dynamodb, test_table_s):
```

Comment:
Same comment on the issue reproduced here too, please.

Reply:
Done !